macros/src/main/scala/io/udash/macros/PropertyMacros.scala

```diff
@@ -217,7 +218,12 @@ class PropertyMacros(val ctx: blackbox.Context) extends AbstractMacroCommons(ctx
         members.map {
           case (name, returnTpe) =>
             val reifiedReturnTpe = reifySeqTpe(returnTpe)
-            q"""properties(${name.toString}) = implicitly[$PropertyCreatorCls[$reifiedReturnTpe]].newProperty(null.asInstanceOf[$reifiedReturnTpe], this)"""
+            val reifiedCreatorTpe = getType(
+              if (tpe <:< SeqTpe) tq"$SeqPropertyCreatorCls[${tpe.typeArgs.head}]"
```

Comment:
When accessing `typeArgs`, beware of:
* type aliases
* classes that extend `Seq` but don't have the same parameterization scheme, e.g. `class WeirdSeq[A,B] extends Seq[B]` or `class IntSeq extends Seq[Int]` or `class SeqSeq[T] extends Seq[Seq[T]]`